### PR TITLE
Update Gatby Version for v3 Compatibility

### PR DESCRIPTION
**Pull request by moonmeister**
Should come with a major version bump.

### CODE REVIEW

**Comment by moonmeister** [comment]
@axe312ger I see you don't seem to be very busy on this repo. If you need help I'm happy to help merge PRs and publish releases.

**Comment by axe312ger** [comment]
As the plugin should work with v2 and v3, could you please allow both versions?

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "size-plugin": "^1.2.0"
   },
   "peerDependencies": {
-    "gatsby": "^2.0.0"
+    "gatsby": "^3.0.0"
```
Comment (axe312ger) [inline package.json]:
```suggestion
    "gatsby": "^2.0.0 || ^3.0.0"
```

**Review by axe312ger — CHANGES_REQUESTED** [review]

**Comment by moonmeister** [comment]
Good call. Merged.

**Comment by axe312ger** [comment]
published as 2.0.1

**Pull request merged**